src/ui/SearchInterface/SearchInterface.ts

```diff
@@ -1030,17 +1030,23 @@ export class SearchInterface extends RootComponent implements IComponentBindings
 
     $$(this.element).on(QueryEvents.doneBuildingQuery, (e, args: IDoneBuildingQueryEventArgs) => {
       if (!args.queryBuilder.containsEndUserKeywords()) {
-        this.logger.info('Query cancelled by the Search Interface', 'Configuration does not allow empty query', this, this.options);
-        args.cancel = true;
-        this.queryStateModel.reset();
-
-        new InitializationPlaceholder(this.element)
-          .withEventToRemovePlaceholder(QueryEvents.newQuery)
-          .withFullInitializationStyling()
-          .withVisibleRootElement()
-          .withPlaceholderForFacets()
-          .withPlaceholderForResultList()
-          .withWaitingForFirstQueryMode();
+        const lastQuery = this.queryController.getLastQuery();
+        if (Utils.isNonEmptyString(lastQuery.q)) {
```

Comment:
Could move the `.q` portion to the `lastQuery` variable and then just check `lastQuery` instead of `lastQuery.q`.